File: JAVA/Daily Flash Codes/Week3/Day2_Jan28/code/prog3.java

import java.util.*;

class Program {
	public static void main(String args[]) {
		Scanner sc = new Scanner(System.in);
		int current , voltage, res;
		System.out.print("Enter current: ");
		current = sc.nextInt();
		
		System.out.print("Enter resistace: ");
		res = sc.nextInt();

		voltage = current * res;
		System.out.println("Voltage: "+voltage);

	}
}

File: C/Daily Flash/Week3/Day2_Jan28/code/prog3.c
/*
 * Program 3: Write a Program to Implement Ohms Law.
{Note: V = I*R, where, v is voltage, I is current & R is resistance}
Input:
I = 10 amp
R = 5
Output: Voltage V = 50.
*/

#include <stdio.h>

void main() {
	int current, resistance , voltage;
	printf("Enter current: ");
	scanf("%d", &current);

	printf("Enter resistance: ");
	scanf("%d", &resistance);
	
	voltage = current * resistance;
	printf("Voltage: %d V\n",voltage);
}
